src/components/SelectionList/index.js

```diff
@@ -1,9 +1,9 @@
 import React, {forwardRef, useEffect, useState} from 'react';
 import {Keyboard} from 'react-native';
-import BaseSelectionListRadio from './BaseSelectionListRadio';
+import BaseSelectionList from './BaseSelectionList';
 import * as DeviceCapabilities from '../../libs/DeviceCapabilities';
 
-const SelectionListRadio = forwardRef((props, ref) => {
+const SelectionList = forwardRef((props, ref) => {
```

Comment:
FYI: When using forwardRef you have to declare `defaultProps` again, [otherwise they won't be considered](https://github.com/facebook/react/issues/16653#issuecomment-564423981). But if you use prop destructuring with default values in your `BaseSelectionList` component you don't need to worry about this.